samza-core/src/main/scala/org/apache/samza/config/StorageConfig.scala

```diff
@@ -66,6 +78,18 @@ class StorageConfig(config: Config) extends ScalaMapConfig(config) with Logging
     systemStreamRes
   }
 
+  //Given a system, this method returns the accesslog system stream
```

Comment:
Documentation: Update comment ("Returns the accesslog stream name given a changelog stream name").